sources/packages/web/src/components/students/StudentDocumentList.vue

```diff
@@ -36,8 +36,8 @@
           @click="fileUtils.downloadStudentDocument(studentDocument)"
         >
           <span class="mr-4">
-            <font-awesome-icon :icon="['far', 'file-alt']"
-          /></span>
+            <v-icon icon="fa:far fa-file-alt" size="20"></v-icon
```

Comment:
Please see previous comments on the size.

Reply:
pls see the above comment